api/RestfulService.php

```diff
@@ -53,7 +53,7 @@ public static function flush() {
 	 */
 	public static function set_default_curl_option($option, $value) {
 		Deprecation::notice('3.2', 'Use the "RestfulService.default_curl_options" config setting instead');
-		Config::inst()->update('RestfulService', 'default_curl_options', array($option => $value));
+		Config::inst()->update(get_called_class(), 'default_curl_options', array($option => $value));
```

Comment:
I agree though that this API should be more consistent ;) Here's the class/file I'm looking at which is returned by `static::config()` calls: https://github.com/silverstripe/silverstripe-framework/blob/3/core/Config.php#L859